bin/run

```diff
@@ -125,59 +150,87 @@ if config.plaintext:
             sess.run(output, {input: input_data}, tag='prediction')
 
             if x % 10 == 0:
-                print('ran', x, 'epochs')
+                print('ran', x, 'iterations')
 
-        print('ran', x, 'epochs')
+        print('ran', x, 'iterations')
 
 
 else:
     print('running prediction securely')
     c = tfe.convert.convert.Converter()
     y = c.convert(graph_def, tfe.convert.register(), 'input-provider', inputs)
 
-    pls = []
-    for i in range(len(input_spec)):
-        if i == 0:
-            name = "private-input/api/{}:0".format(i)
-        else:
-            name = "private-input_{}/api/{}:0".format(i, i)
-        pls.append(
-            tf.get_default_graph().get_tensor_by_name(name)
-        )
+    if config.data_prep_path:
```

Comment:
This if branch is identical to what comes later. Consider 
‘’‘
if not config.data_prep_path:
    # handle placeholders logic from lines 190-199
with tfe.Session():
    # etc.
‘’’